src/applications/personalization/rated-disabilities/sass/rated-disabilities.scss

```diff
@@ -10,13 +10,37 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 .rated-disability-item-container {
     background-color: #f1f1f1;
     margin: 1em 0;
-    padding: 1em;
+    padding: 0.5em 1em 1em;
+
+    &:first-of-type {
+        border-top: 1px solid $color-base;
+    }
 }
 
 .disability-item-label {
     font-weight: 700;
+}
+
+.disability-item-name {
+    font-weight: 700;
+    margin: 3px 0 0 0;
+    font-size: 1.75rem;
+}
+
+.disability-item-percentage {
+    font-weight: 700;
     margin: 0;
+    font-size: 2rem;
+    text-align: right;
 }
+
 .disability-item-desc {
     margin: 0;
+}
+
+.disability-item-spacer {
+
+    @media screen and (max-width: $medium-screen) {
+        display: none;
```

Comment:
We have display classes that can be combined with our responsive prefixes [here](https://design.va.gov/utilities/display)